xml/System.Collections.Generic/HashSet`1+AlternateLookup`1.xml

```diff
@@ -31,8 +31,8 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <typeparam name="TAlternate">To be added.</typeparam>
-    <summary>To be added.</summary>
+    <typeparam name="TAlternate">The alternate type of instance for performing lookups.</typeparam>
+    <summary>Provides a type that may be used to perform operations on a <see cref="T:System.Collections.Generic.HashSet`1" /> using a <typeparamref name="TAlternate" /> instead of a <typeparamref name="T" />.</summary>
```

Comment:
```suggestion
    <summary>Provides a type that can be used to perform operations on a <see cref="T:System.Collections.Generic.HashSet`1" /> using a <typeparamref name="TAlternate" /> instead of a <typeparamref name="T" />.</summary>
```